community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/labelscan/LuceneLabelScanStore.java

```diff
@@ -36,16 +35,19 @@
 
 public class LuceneLabelScanStore implements LabelScanStore
 {
-    private final LabelScanIndex luceneIndex;
+    private final LuceneLabelScanIndexBuilder indexBuilder;
+    private volatile LabelScanIndex luceneIndex;
```

Comment:
It's a field in the `LuceneLabelScanStore` so I hope it has to do with lucene :)